confirmation/models.py

```diff
@@ -43,7 +43,8 @@ def generate_key() -> str:
 
 ConfirmationObjT = Union[MultiuseInvite, PreregistrationUser, EmailChangeStatus]
 def get_object_from_key(confirmation_key: str,
-                        confirmation_type: int) -> ConfirmationObjT:
+                        confirmation_type: int,
+                        activate_object: bool=True) -> ConfirmationObjT:
```

Comment:
I don't understand when `activate_object` won't be True, if the default is True and we never pass False explicitly?

Reply:
We do in `registration.py` https://github.com/zulip/zulip/pull/14584/files#diff-e8571ba96eb5129e439cd7d7843dedd3